modules/svg/data.js

```diff
@@ -314,6 +314,27 @@ export function svgData(projection, context, dispatch) {
         return (new DOMParser()).parseFromString(textdata, 'text/xml');
     }
 
```

Comment:
There's a convention within iD of leaving two spaces between function calls, so please add another space here!

Reply:
Done!